src/main/java/com/minecolonies/coremod/util/constants/RenderUtils.java

```diff
@@ -0,0 +1,144 @@
+package com.minecolonies.coremod.util.constants;
+
+import com.minecolonies.coremod.colony.ColonyManager;
+import com.minecolonies.coremod.colony.ColonyView;
+import com.minecolonies.coremod.configuration.Configurations;
+import com.minecolonies.coremod.util.BlockPosUtil;
+import com.minecolonies.structures.helpers.Settings;
+import com.minecolonies.structures.helpers.Structure;
+import com.minecolonies.structures.lib.ModelHolder;
+import net.minecraft.block.Block;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.entity.EntityPlayerSP;
+import net.minecraft.client.multiplayer.WorldClient;
+import net.minecraft.client.renderer.block.model.IBakedModel;
+import net.minecraft.entity.item.EntityEnderCrystal;
+import net.minecraft.init.Blocks;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.util.math.BlockPos;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * I need that to calculate the circle, if someone knows a better library tell me.
+ */
+import static sun.awt.geom.Curve.next;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Use classes from the Java API instead of Sun classes. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1191)